projects/igniteui-angular/src/lib/grid/grid.component.ts

```diff
@@ -1224,6 +2350,15 @@ export class IgxGridComponent implements OnInit, OnDestroy, AfterContentInit, Af
         this.refreshSearch();
     }
 
+    /**
+     * Removes the `IgxGridRowComponent` and the corresponding data record
```

Comment:
Removes the `IgxGridRowComponent` and the corresponding data record by primary key. Requires that the `primaryKey`  property is set.